sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/optimizer/NormalizeFloatingNumbers.scala

```diff
@@ -56,10 +56,6 @@ import org.apache.spark.sql.types._
 object NormalizeFloatingNumbers extends Rule[LogicalPlan] {
```

Comment:
Does it also mean `This batch must be executed after the `RewriteSubquery` batch, which creates joins.` is not definitely true now?

Reply:
ah yea, actually it's better to execute it before the `RewriteSubquery`

Reply:
I see. Makes sense.